### PR TITLE
Adding updateCell method

### PR DESCRIPTION
**Pull request by anhnt**
This method allow updating cell data manually.
Param:
+ rowIndex: the row index of cell
+ fieldName: column field name of cell
+ fieldValue: new value to update

### CODE REVIEW

**Comment by djhvscf** [comment]
Please include the docs updated and also you have to call the initSort method before the initBody method

File: src/bootstrap-table.js [hunk 1]
```diff
@@ -1855,7 +1855,14 @@
...
     };
-
+    
+    BootstrapTable.prototype.updateCell = function (params) {
+        if (!params.hasOwnProperty('rowIndex') || !params.hasOwnProperty('fieldName') || !params.hasOwnProperty('fieldValue')) {
+            return;
+        }
+        this.data[params.rowIndex][params.fieldName] = params.fieldValue;
+        this.initBody(true);
+    };
```
Comment (djhvscf) [inline src/bootstrap-table.js]: Here is missing a space between the functions

File: src/bootstrap-table.js [hunk 2]
```diff
@@ -1833,7 +1833,7 @@
         }
         return rows;
     }
-
+
```
Comment (djhvscf) [inline src/bootstrap-table.js]: Delete this double space

**Comment by wenzhixin** [comment]
This PR needs to rebase.

**Comment by djhvscf** [comment]
I will close this PR and include in my current PR.. ok? @wenzhixin

**Comment by anhnt** [comment]
Thanks @djhvscf . Can you give me the link to your PR. I'm busy these days and cannot update the code following your comment :)

**Comment by djhvscf** [comment]
@anhnt  https://github.com/wenzhixin/bootstrap-table/pull/885

**Comment by anhnt** [comment]
Nice. Thanks again!

**Comment by wenzhixin** [comment]
Merged! Thanks, @anhnt , @djhvscf

**Comment by anhnt** [comment]
Hi guys, do we need pushing "updateCell" method name into allowedMethods array?

**Comment by djhvscf** [comment]
Yes.. Can you add that?

**Comment by djhvscf** [comment]
Before the release we have to add that.. @wenzhixin

**Comment by wenzhixin** [comment]
Added

**Pull request closed without merging**